modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/js/edit_cp_definition_visibility.js

```diff
@@ -29,6 +29,8 @@ function handleEvent({
 
 	return AdminCatalogResource.updateProduct(productId, formattedData).finally(
 		() => {
+			window.location.reload();
+
```

Comment:
😮 ... is this needed for? But also if you reload the page I don't know if the next lines will be executed correctly